### PR TITLE
chore(docs): add GITGUARDIAN_API_URL to pre-receive hook template

### PR DESCRIPTION
**Pull request by CharlesB2**

### CODE REVIEW

File: doc/pre-receive.sample [hunk 1]
```diff
@@ -9,7 +9,10 @@
 # either set an environment variable machine wide `GITGUARDIAN_API_KEY`
 # or uncomment and insert your key in the following line
 
-#export GITGUARDIAN_API_KEY=<INSERT YOUR KEY>
+# export GITGUARDIAN_API_KEY=<INSERT YOUR KEY>
+
+# (set this if you are using a self-hosted GitGuardian instance)
```
Comment (agateau-gg) [inline doc/pre-receive.sample:14]: nit: wording.
```suggestion
# Set this if you use a self-hosted GitGuardian instance
```

File: doc/pre-receive.sample [hunk 2]
```diff
@@ -9,7 +9,10 @@
...
+# export GITGUARDIAN_API_URL="https://dashboard.gitguardian.mycorp.local/exposed/"
```
Comment (agateau-gg) [inline doc/pre-receive.sample:15]: Is the "exposed/" part always there for self-hosted instances? If not I would use a simpler URL for the example.

Comment (CharlesB2) [reply doc/pre-receive.sample:15]: Yes, after checking the `exposed` part is necessary for self-hosted

Comment (agateau-gg) [reply doc/pre-receive.sample:15]: OK, all good then :+1:

**Review by agateau-gg — APPROVED** [review]

**Pull request merged**